Helper/ConfigHelper.php

```diff
@@ -126,6 +127,15 @@ public function isEventCartViewEnabled()
         return $cart_view_code == 1;
     }
 
+    public function isCartAutoCouponEnabled()
+    {
+        $value = $this->_scopeConfig->getValue(self::CART_AUTO_COUPON_ENABLED);
+        if ($value === null) {
+            return true;
```

Comment:
Does this mean that the default value is "true"?

Reply:
exactly

Reply:
yes, we wanted to have it enabled by default

Reply:
OK, I'm thinking about this again, and since this new code changes the behavior of the website, it would actually be better to have it turned off by default, so that site admins aren't surprised by new functionality when they didn't mean to activate it.